app/models/image.rb

```diff
@@ -13,21 +13,6 @@ class Image < ApplicationRecord
   belongs_to :document
   belongs_to :blob, class_name: "ActiveStorage::Blob"
 
-  validates :width,
-            numericality: { only_integer: true, greater_than_or_equal_to: WIDTH }
-  validates :height,
-            numericality: { only_integer: true, greater_than_or_equal_to: HEIGHT }
-  validates :crop_x,
-            numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :crop_y,
-            numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :crop_width,
-            numericality: { only_integer: true, greater_than_or_equal_to: WIDTH }
-  validates :crop_height,
-            numericality: { only_integer: true, greater_than_or_equal_to: HEIGHT }
-
-  validates_with ImageAspectRatioValidator
-
```

Comment:
Should it still do the validation?

Reply:
Good question - I think I addressed this in the second paragraph of the commit. Because we're confident the UI will enforce the image constraints, we don't feel it adds value to have these validations here. The other possibility is a programmer might try to set these values when debugging something, but we think this is unlikely for images ;-).

Reply:
I think the best rule to follow is to never trust the browser, however, as you are working out the height based on the width - I think it might be fine.

Reply:
I think you've got @kevindew 's thoughts on this ;-). You're right that we should ideally have a validator on the critical path here; we do this for documents, because we anticipate the user doing something wrong, but as you say, the use shouldn't be able to specify an invalid crop.